src/refresh/model_iqm.c

```diff
@@ -482,30 +482,30 @@ int MOD_LoadIQM_Base(model_t* model, const void* rawdata, size_t length, const c
 
 		if (vertexArrayFormat[IQM_BLENDWEIGHTS] != -1)
 		{
-			CHECK(iqmData->blend_weights = (float*)MOD_Malloc(header->num_vertexes * 4 * sizeof(float)));
+			CHECK(iqmData->blend_weights = MOD_Malloc(header->num_vertexes * 4 * sizeof(byte)));
 		}
 	}
 
 	if (header->num_joints)
 	{
-		CHECK(iqmData->jointNames = (char*)MOD_Malloc(joint_names));
-		CHECK(iqmData->jointParents = (int*)MOD_Malloc(header->num_joints * sizeof(int)));
-		CHECK(iqmData->bindJoints = (float*)MOD_Malloc(header->num_joints * 12 * sizeof(float))); // bind joint matricies
-		CHECK(iqmData->invBindJoints = (float*)MOD_Malloc(header->num_joints * 12 * sizeof(float))); // inverse bind joint matricies
+		CHECK(iqmData->jointNames = MOD_Malloc(joint_names));
```

Comment:
this just reminded me, on my local branch I changed jointNames to be an array of `char[MAX_QPATH]` so that I can easily index the joint names at runtime. Currently, you need to loop over the string via strlen which is pretty inefficient.. not that it totally matters I guess since you shouldn't be looping through joint names every frame at runtime, but just a random note that I think a single long string of "name data" is a bit harder to parse through than just fixed-length string indices.